prow/cluster/jobs/istio-releases/daily-release/istio-releases.daily-release.release-1.1.yaml

```diff
@@ -0,0 +1,158 @@
+job_template: &job_template
```

Comment:
nit: `job_template` is a bit of a confusing name since this template isn't used for all jobs.

Reply:
Ack. I am sure we need to overhaul our prow config some day.